lib/init.coffee

```diff
@@ -32,6 +35,11 @@ module.exports =
       type: 'string'
       default: 'luacheck'
       description: 'The executable path to luacheck.'
+    standard:
+      type: 'array'
+      default: []
+      description: "List of comma separated standards.
+        eg. `min, busted`"
```

Comment:
Ah, if that's the case use string concatenation since right now you have a bunch of whitespace in the string.